compiler/rustc_infer/src/infer/relate/combine.rs

```diff
@@ -177,6 +179,8 @@ impl<'tcx> InferCtxt<'tcx> {
     where
         R: PredicateEmittingRelation<InferCtxt<'tcx>>,
     {
+        use InferConst::*;
```

Comment:
personally not a big fan of this commit, but :shrug: I don't care much either way

Reply:
I'd rather not use glob imports like this, so I'd prefer if we remove it.